editoast/src/views/infra/auto_fixes.rs

```diff
@@ -81,9 +81,19 @@ async fn fix_infra(infra_cache: &mut InfraCache) -> Result<Vec<Operation>> {
         }
     }
 
-    let operation_results = all_fixes.iter().map_into().collect();
+    let cache_operations = all_fixes
+        .iter()
+        .map(|operation| match operation {
+            Operation::Create(railjson) => CacheOperation::Create(railjson.deref().clone().into()),
+            Operation::Update(_) => unimplemented!("not possible at the moment, wait for refactor"),
+            Operation::Delete(delete_operation) => CacheOperation::Delete(ObjectRef::new(
+                delete_operation.obj_type,
+                delete_operation.obj_id.clone(),
+            )),
```

Comment:
```suggestion
            Operation::Delete(delete_operation) => {
                CacheOperation::Delete(delete_operation.clone().into())
            }
```

Reply:
Done [here](https://github.com/osrd-project/osrd/pull/5924/files#diff-7121059a1a4c96a8a6c3318b0df4199fd8cf51e08ea86de2e2c79d437f229869R90). Thanks for the hint.